src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -53,10 +53,8 @@ values of the `OtlpExporterOptions`
 
 ## Special case when using insecure channel
 
-If your application is [.NET Standard
-2.1](https://docs.microsoft.com/dotnet/standard/net-standard) or above, and you
-are using an insecure (http) endpoint, the following switch must be set before
-adding `OtlpExporter`.
+If your application is targeting .NET Core 3.1, and you are using an insecure
```

Comment:
Just to confirm, only ".NET Core 3.1" not ".NET Core 3.1 or above"?

Reply:
Seems that [this doc](https://docs.microsoft.com/en-us/aspnet/core/grpc/troubleshoot?view=aspnetcore-5.0#call-insecure-grpc-services-with-net-core-client) has confirmed that only 3.x is affected。

Reply:
Correct and I tested it as well

Reply:
@pellared thanks for confirmation!